src/server/connection.py

```diff
@@ -120,15 +128,33 @@ def __str__(self) -> str:
 
 
 class PeerConnections:
-    def __init__(self, all_connections: List[ChiaConnection] = []):
+    def __init__(
+        self,
+        local_type: NodeType,
+        root_path,
+        config: Dict,
+        all_connections: List[ChiaConnection] = []
+    ):
         self._all_connections = all_connections
         # Only full node peers are added to `peers`
         self.peers = Peers()
```

Comment:
Can we remove the Peers class? Or put it in the introducer instead. It is making the code here more confusing